src/browse.js

```diff
@@ -44,7 +47,7 @@ var browse = {
     },
 
     fetchRepos:(keyword, widgetId) => {
-        var url = "https://api.github.com/search/repositories?q=topic:" + keyword + "&sort=stars&per_page=1000";
+        var url = "https://api.github.com/search/repositories?q=user:azure+user:microsoft+user:azure-samples+topic:" + keyword + "&sort=stars&per_page=1000";
```

Comment:
Can we stay away from the additional hard code here? Maybe take an input array parameter and we can use the same hard-code array once, like src/index.js, line 9? And eventually such hard-code array should become configurable.